swift/LottieRenderer.swift

```diff
@@ -30,6 +30,7 @@ public class LottieRenderer {
         self.canvas = Canvas(size: size, buffer: buffer, stride: stride)
 
         let picture = lottie.animation.getPicture()
+        picture.resize(canvas.size)
```

Comment:
Also, I've moved the resize step to the renderer initialiser too. Now that all we're dealing with is transforms below, it makes sense to set the size once and then not have to worry about it later. Will help us further move towards being stateless as well.